packages/blade/src/components/Button/BaseButton/BaseButton.tsx

```diff
@@ -432,13 +436,15 @@ const _BaseButton: React.ForwardRefRenderFunction<BladeElementRef, BaseButtonPro
       onPointerDown={onPointerDown}
       onPointerEnter={onPointerEnter}
       onKeyDown={onKeyDown}
+      onTouchStart={onTouchStart}
+      onTouchEnd={onTouchEnd}
       type={type}
       borderWidth={borderWidth}
       borderRadius={borderRadius}
       motionDuration={motionDuration}
       motionEasing={motionEasing}
       {...metaAttribute({ name: MetaConstants.Button, testID })}
-      {...styledProps}
```

Comment:
so we shall never use `...styledProps` now and use `...getStyledProps(rest)` always?

Reply:
Mostly yes. If you know that props are filtered then you can still do `...styledProps` (e.g. if I am filtering it on top component somewhere and pass it to lets say `StyledButton`, then I don't have to filter them in StyledButton again.

But yes in general, it's best to do that anyway.